src/status_im/ui/screens/desktop/main/chat/views.cljs

```diff
@@ -135,16 +135,19 @@
           current-public-key (re-frame/subscribe [:get-current-public-key])]
       [react/view {:style styles/messages-view}
        [react/scroll-view {:scrollEventThrottle    16
+                           :headerHeight styles/messages-list-vertical-padding
+                           :footerWidth styles/messages-list-vertical-padding
+                           :enableArrayScrollingOptimization true
                            :on-scroll              (fn [e]
                                                      (let [ne (.-nativeEvent e)
                                                            y (.-y (.-contentOffset ne))]
-                                                       (when (zero? y)
+                                                       (when (<= y 0)
```

Comment:
Negative y values are possible when view flicked down from the top most position. In terms of current implementation it means that user wants to load more messages from the history.